src/vaadin-menu-bar-interactions-mixin.html

```diff
@@ -205,10 +226,12 @@
       return -1;
     }
 
+    /** @private */
```

Comment:
I did not add types for `vaadin-menu-bar-submenu` element because it's not currently supposed to be used directly by developers. So this is marked as `@private` despite we use it in unit tests quite a lot.